src/test/java/org/terracotta/statistics/derived/histogram/ExponentialHistogramTest.java

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2015 Terracotta, Inc..
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics.derived.histogram;
+
+import static org.hamcrest.core.Is.is;
```

Comment:
It's weird that the static imports are above the others

Reply:
Will fix.